ts_sim: 0.01
ts_ctl: 0.01
ts_viz: 0.05

# time_factor: 1.0  # only 1x simulation is supported now

has_downwash: True
has_motor_model: True
has_battery: True

# enu frame
qd_init_states:
  - name: "fhnp"
    init_pos:
      - 1.0 # px
      - 1.0 # py
      - 5.0 # pz
